### DISCUSSION TITLE
Why is the preprocessed data shown in the web server different with the one I downloaded from the preprocessed folder?

### DISCUSSION THREAD

**Discussion by dahui-y** [nextpyp/.github#133] [Q&A] [answered]
Hi!

I've made the preprocessing to the raw data (i.e., stage 2). Then, I would like to download the preprocessed data and visualize it. However, the preprocessed tomogram I downloaded looks more blur than the one shown on the web. Therefore, I am wondering how I can download the one shown on the web. To make clear what my question is, I attached the corresponding pictures shown below.

The tomogram shown on the web,
<img width="277" alt="Screenshot 2024-12-16 at 10 49 22" src="https://github.com/user-attachments/assets/cbe76752-3496-4efb-a882-24dfc37dd32d" />

The tomogram I downloaded from the preprocessed folder 'mrc' is shown by Imod,
<img width="543" alt="Screenshot 2024-12-16 at 10 52 06" src="https://github.com/user-attachments/assets/aee5f4a4-58ee-468c-a788-1eb93001c483" />

**Comment by abartesaghi** [comment, marked as answer]
This is because the image on the web is generated by averaging two consecutive slices from the .mrc tomogram file, which gives better contrast. You should be able to replicate this behavior in IMOD using the [Slicer](https://bio3d.colorado.edu/imod/doc/3dmodHelp/slicer.html) tool.
